src/main/java/frc/robot/Constants.java

```diff
@@ -154,4 +154,9 @@ public class RobotVersion2024 implements RobotVersionConstants {
   }
 
   public static final RobotVersion defaultRobotVersion = RobotVersion.v2024;
+
+  /** <b> DETAILED EXPLANATION </b> */
+  public static int Explanation() {
+    return 1;
+  }
```

Comment:
What is this?

Reply:
I Created a function called explanation in each class. in the explanation file people will get a clear explanation of how things work based off of the java doc of that explanation

Reply:
Why is there a function rather a direct comment block with the class and comment blocks for each of the functions? Why use this arbitrary `Explanation` function?

Reply:
I didnt want to make the explanation for the class to long

Reply:
I think you should focus more on comment blocks, rather than this `Explanation` class system. I would revert commit: 19b043aa82a8bf173b75094d610b96a030392206

Reply:
Also We need to have explanations for things like init Sendable and command schedule and auto etc

Reply:
Those can be done for the `initSendable` functions, individually

Reply:
What about command Scheduler etc. I want to make it so that each student who wants to code after me knows exactly where to look for stuff. I cant java doc the init sendable stuff really and command scheduler stuff

Reply:
Take this to the main comment block. I'll respond there.